Modules/Sources/WordPressShared/URL+Helpers.swift

```diff
@@ -2,18 +2,18 @@ import Foundation
 import MobileCoreServices
 import UniformTypeIdentifiers
 
-extension URL {
+public extension URL {
```

Comment:
I started by compiling this as `internal` with the intention of only making `public` what blocked compilation. But after a few build iterations, I kept getting access control errors. So I just made everything `public` and accepted the chance that some methods or vars might be unused.